Improve docs (#428)
qlik-oss/sense-installer: docs/index.md

# Overview

The Qlik Sense on Kubernetes Operator CLI (`qliksense`)  facilitates:

- Installation of QSEoK
- Installation of Qliksense operator to manage the QSEoK installation
- Air gapped installation of QSEoK
- Cluster configuration management
- Pre-flight and Post-flight environment and configuration checks

The Qlik Sense on Kubernetes Operator CLI provides an imperative interface to many of the configurations that need to be applied against the declarative structure described in the [qliksense-k8s](https://github.com/qlik-oss/qliksense-k8s) repository

> To get start quickly go to the Getting Started page.
> 
> Otherwise, proceed to learn more about the internal workings of the Qlik Sense on Kubernetes Operator in the concepts page.
> 
> ! info ""
>     This is a technology preview that uses Qlik modified [kustomize](https://github.com/qlik-oss/kustomize) for Kubernetes manifests on [qliksense-k8s](https://github.com/qlik-oss/qliksense-k8s) repository

Reviewer: ```suggestion
To get start quickly go to the [Getting Started page](getting_started.md).

To learn more about the internal workings of the Qlik Sense on Kubernetes Operator, go to [How CLI works](concepts.md).

!!! info ""
    This is a technology preview that uses Qlik modified [kustomize](https://github.com/qlik-oss/kustomize) for Kubernetes manifests on [qliksense-k8s](https://github.com/qlik-oss/qliksense-k8s) repository

```
